sw/device/tests/BUILD

```diff
@@ -102,7 +106,13 @@ opentitan_test(
 opentitan_test(
     name = "aes_idle_test",
     srcs = ["aes_idle_test.c"],
-    exec_env = EARLGREY_TEST_ENVS,
+    exec_env = dicts.add(
```

Comment:
Didn't know about `dict.add`. This is nice!

Reply:
It was a suggestion from @cfrantz. I had also used these before. There are a few other handy ones, like paths. See https://github.com/bazelbuild/bazel-skylib.